.tav.yml

```
@@ -189,6 +189,12 @@ elasticsearch:
 handlebars:
   versions: '*'
   commands: node test/instrumentation/modules/handlebars.js
+jade:
+  versions: '*'
+  commands: node test/instrumentation/modules/jade.js
+pug:
+  versions: '*'
```

As @Qard mentioned, you should just be able to exclude version 2.0.0 of pug from this list by doing so:

```suggestion
  versions: '<2.0.0 >2.0.0'
```

---

If you want to run this locally first to see which versions are supported, you can run this command:

```sh
TAV=pug ./node_modules/.bin/tav --compat
```

This will install all versions in the allowed range and print out if they are failing or not and allow you to see if there's more versions than just 2.0.0 that are failing. It will not print out the error though. To view that, just replace `--compat` with `--quiet`.

---

I want to test it locally but got error `zsh: no such file or directory: ./node_modules/.bin/tav` .

---

oh, I see. here it is https://github.com/watson/test-all-versions

---

the tav binary is a devDependency and will be installed automatically when you run `npm install`. If you installed using `npm install --production` you'll not have it